frontend/packages/component-library/package.json

```diff
@@ -302,6 +302,7 @@
     "@types/node": "^22.9.0",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
+    "classnames": "^2.5.1",
```

Comment:
Not sure I understand. Why we move `classnames` to devDependencies?

Reply:
Gotcha, thank you for explaining!
I wonder if we can keep this code and (should?) also update it in dashboard since it's minor version difference, just to keep dashboard updated?

Reply:
No no, your approach is awesome I think and I agree with it!

I meant if just on top of what we already have here should we also update apps classnames version (make a bump) basically just to update it

Reply:
Yes we should bump it, but probably out of scope for this PR